Balancer-Bots/Large-BAL-Transfer/src/agent.ts

```diff
@@ -0,0 +1,73 @@
+import { Finding, HandleBlock, BlockEvent, getEthersProvider } from "forta-agent";
+import { BigNumber } from "bignumber.js";
+import { providers, utils, Contract } from "ethers";
+import { NetworkManager } from "forta-agent-tools";
+import { NetworkData, SmartCaller, toBn } from "./utils";
+import { EVENT, TOKEN_ABI } from "./constants";
+import CONFIG from "./agent.config";
+import { createFinding } from "./findings";
+
+BigNumber.set({ DECIMAL_PLACES: 18 });
+
+const networkManager = new NetworkManager<NetworkData>(CONFIG);
+
+export const initialize = (networkManager: NetworkManager<NetworkData>, provider: providers.Provider) => {
+  return async () => {
+    await networkManager.init(provider);
+  };
+};
+
+export const provideHandleBlock = (
+  provider: providers.Provider,
+  networkManager: NetworkManager<NetworkData>
```

Comment:
Minor, but it would be good to follow the same parameter order as `provideInitialize`.